### DISCUSSION TITLE
Cycle 24 started

### DISCUSSION THREAD

**Discussion by MwithM** [bisq-network/bisq#5414] [Show and tell]
- The [rate](https://github.com/bisq-network/compensation/issues/830) for compensation requests on Cycle 23 is 2USD/BSQ, but it could be modified to 2.18USD/BSQ. Please vote if you want to remove deciding the BSQ rate a week before end of proposal phase.
- [Deadline](https://calendar.google.com/calendar/embed?src=adfiuiokq9jtac8t9keomc00b0%40group.calendar.google.com&ctz=Europe%2FMadrid) to submit compensation requests for review is 3 May.
- BSQ trading fee has been [updated](https://github.com/bisq-network/proposals/issues/325).

**Comment by pazza83** [comment]
Hi @MwithM the Deadline link goes to a google doc that is not available to view?

**Reply by MwithM** [reply]
Thanks, it has been edited.
